### DISCUSSION TITLE
Deepin v20 (1001) Auto merge

### DISCUSSION THREAD

**Discussion by yarecco** [linuxdeepin/developer-center#2966] [Features Request & Ideas | 特性请求 & 头脑风暴]
Just give it back
![909-menu](https://user-images.githubusercontent.com/29832803/91502501-8030b380-e8c8-11ea-857e-404990a50488.jpg)

**Comment by XMuli** [comment]
Is it the latest version? Then this is missing?  Please follow the format to submit issues

**Comment by yarecco** [comment]
Yes it is missing in latest update (1001)

**Comment by deepin-CD** [comment]
Thanks for your suggestion.
We will seriously consider it.

**Comment by meiyixiang** [comment]
This is the deepin 15 desktop automatic archiving function, right? This feature is not available in the current version.

**Comment by meiyixiang** [comment]
I will feedback your suggestions to the product team
